### PR TITLE
Add MasterCard Argentina

### PR DESCRIPTION
**Pull request by fainpablo**

### CODE REVIEW

**Comment by phallobst** [comment]
This one should go into "Payments" as well.

**Comment by fainpablo** [comment]
Moved, thanks.

**Comment by Carlgo11** [comment]
Is it just me or is there no login button?

File: _data/payments.yml [hunk 1]
```diff
@@ -82,6 +82,12 @@ websites:
       - proprietary
     exception: "Software 2FA is only available for Swedish citizens with BankID."
 
+  - name: MasterCard Argentina
+    url: https://www.masterconsultas.com.ar
+    img: mastercard.png
+    twitter: MastercardArg
+    facebook: MastercardArgentina
```
Comment (phallobst) [inline _data/payments.yml:95]:
```suggestion
    facebook: MastercardArgentina
    lang: es
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Login is in the top right hand corner:
image

**Comment by fainpablo** [comment]
> Is it just me or is there no login button?

Top right corner. "Acceda a sus cuentas on-line" and then you have to select the type of customer you are.


Screen Shot 2020-02-19 at 16 21 12

**Comment by Carlgo11** [comment]
And what can you do when logged in?

**Comment by fainpablo** [comment]
> And what can you do when logged in?

You can manage your credit cards no matter which bank has issued them. Issue a new plastic, request cards for additional holders, cancel your card, pay bills, etc. Same as Visa Argentina.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**